department-of-veterans-affairs/vets-api
# Reset session cookies before logging in

## Description of change
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/15024
If a user attemped to log in using MHV with a current SSO cookie they would see "unable to process SAML request"
So, clear sso cookies before we redirect to the SAML provider with for all login types for good measure.

## Testing done
Tested locally with id.me that the sso cookie was deleted when the SAML popup opens.

## Testing planned
Test with DS Logon and MHV in stage (someone with a PIV)

testing steps for each SAML provider:
1) log in
2) close all tabs for site, but don't quit browser
3) go back to site, and  attempt to log in again 
4) verify there is no SSO cookie after login popup opens
5) successful login

#### Unique to this PR

- [ ] _Replace this line with individual tasks unique to your PR_

#### Applies to all PRs

- [x] Appropriate logging
- [x] Swagger docs have been updated, if applicable
- [x] Provide link to originating GitHub issue, or connected to it via ZenHub
- [ ] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [ ] Provide which alerts would indicate a problem with this functionality (if applicable)
